src/reducers/locations.js

```diff
@@ -69,18 +69,15 @@ const locationsReducer = (state = {}, action) => {
         const {
           Languages,
           DocumentsInfo,
-          RequiredDocuments,
           RegularSchedules,
           HolidaySchedules,
           EventRelatedInfos,
         } = service;
 
-        if (documents.proofs != null) {
-          // TODO: Update when form support multiple proofs
-          // eslint-disable-next-line prefer-destructuring
-          const proof = { ...RequiredDocuments[0], document: documents.proofs[0] };
+        let { RequiredDocuments } = service;
 
-          RequiredDocuments[0] = proof;
+        if (documents.proofs != null) {
+          RequiredDocuments = documents.proofs.map(p => ({ document: p }));
```

Comment:
Just to make sure: There are no fields on the object other than `document`, that need to be preserved?
(I see the previous code was keeping all the other fields' values, but don't know if there was a reason for that.)

Reply:
Yeah, it's only taking that string on the backend.